inductiva/fluids/scenarios/coastal_area/coastal_area.py

```diff
@@ -120,9 +120,11 @@ def from_random_depths(
             random.uniform(-max_depth, 0),
         ]
 
-        _, _, depths = inductiva.generative.procedural.generate_random_terrain(
-            x_range, y_range, x_num, y_num, corner_values, initial_roughness,
-            roughness_factor, percentile_above_water)
+        depths = inductiva.generative.procedural.generate_random_terrain(
+            x_num, y_num, corner_values, initial_roughness, roughness_factor)
+
+        depths = inductiva.generative.procedural.adjust_terrain_elevation(
```

Comment:
I would drop the words "terrain" and "elevation". I think we are still working at the abstract non-physical level. This should be something like adjust_depth_map_level.

I am not sure if we are going back, but I am even ok with merging this method with the previous, since it requires adding only one parameter and can be fully implemented at the "abstract" / "topological" level, i..e without requiring notion of the physical dimensions.

It's up to you, but I think we should drop "terrain" from the name.